cryptography/go/sdk/README.md

```diff
@@ -0,0 +1,37 @@
+# Dapr secrets management (Dapr SDK)
+
+In this quickstart, you'll create an application that encrypts, and then decrypts, a short message and a large file using the Dapr cryptography APIs (high-level).
+
+Visit the documentation to learn more about the [Cryptography building block](https://docs.dapr.io/developing-applications/building-blocks/cryptography/) in Dapr.
+
+> **Note:** This example uses the Dapr SDK. Using the Dapr SDK, which leverages gRPC internally, is **strongly** recommended when using the high-level cryptography APIs (to encrypt and decrypt messages).
```

Comment:
Can we add a sentence that explains why it's recommended to use the Dapr SDKs with this API? What does it provide for me that I'd have to do myself if I wasn't using and SDK?

Reply:
I have checked other quickstarts and I don't see this mentioned anywhere else. The truth is that implementing this by hand is quite complex as it requires dealing with chunking.